app/views/sessions/new.html.erb

```diff
@@ -1,51 +1,71 @@
 <% content_for :head do %>
   <%= javascript_include_tag "login" %>
+  <%= javascript_include_tag "auth_providers" %>
 <% end %>
 
-<% content_for :heading do %>
-  <h1><%= t ".heading" %></h1>
-<% end %>
+<% content_for :heading_class, "p-0 mw-100" %>
 
-<div id="login_login">
-  <p class='text-muted'><%= t ".no account" %> <%= link_to t(".register now"), user_new_path(:referer => params[:referer]) %></p>
+<% content_for :heading do %>
+  <% if @client_app_name %>
+    <p class="text-center text-muted fs-6 py-2 mb-0 bg-white"><%= t(".login_to_authorize_html", :client_app_name => @client_app_name) %></p>
+  <% end %>
 
-  <%= bootstrap_form_tag(:action => "login", :html => { :id => "login_form" }) do |f| %>
-    <%= hidden_field_tag("referer", h(params[:referer]), :autocomplete => "off") %>
+  <div class="row justify-content-center">
```

Comment:
I'm getting a horizontal scrollbar on narrow screens here without `mx-0`.

Reply:
I am not able to reproduce this.

Can you post a screenshot and/or details on browser/os involved?

Are you suggesting that I should add "mx-0" to the styles of `<div class="row justify-content-center">` element?

Reply:
Actually it's on any screen size.

![image](https://github.com/openstreetmap/openstreetmap-website/assets/4158490/02604f27-6344-4ea5-8b89-698a707019e7)

![image](https://github.com/openstreetmap/openstreetmap-website/assets/4158490/433b0007-a2ee-4dd9-8ef8-89702f9f6210)

Chromium Version 122.0.6261.94 (Official Build) snap (64-bit)
and
Firefox 123.0 on Ubuntu

Reply:
Maybe replacing `row` with `d-flex` is better. With row you have negative margins, then padding inside.

Reply:
Or maybe even

```
<div class="header-illustration new-user-main auth-container mx-auto">
```

instead of

```
<div class="row justify-content-center">
    <div class="header-illustration new-user-main auth-container">
```

Reply:
Found it, actually horizontal scroll bar was always present. mx-0 fixes this